app/views/devise/registrations/new.js.erb

```diff
@@ -0,0 +1,17 @@
+if (typeof displayElementLogin !== undefined) {
+    displayElementLogin = document.getElementById('user-forms')
+    signupForm = "<%= escape_javascript(render partial: 'partials/signup') %>"
+    modalLogin = document.getElementById('modal')
+} else {
+    let signupForm = "<%= escape_javascript(render partial: 'partials/signup') %>"
+    let displayElementLogin = document.getElementById('user-forms')
+    let modalLogin = document.getElementById('modal')
+}
+displayElementLogin.innerHTML = signupForm;
+
+window.onclick = function(event) {
```

Comment:
this function is in booth reg/new.js and sessions/new.js. Can it be moved to application.js?

Reply:
it can, it worked fine